epsagon/events/botocore.py

```diff
@@ -2,7 +2,7 @@
 botocore events module.
 """
 
-# pylint: disable=C0302
+# pylint: disable=C0301, C0302
```

Comment:
Not sure if it's needed at all, but if it is, fix the line length instead of ignoring the linter (unless there is an actual issue).